bears/hypertext/HTMLLintBear.py

```diff
@@ -27,10 +27,17 @@ class HTMLLintBear:
     CAN_DETECT = {'Syntax', 'Formatting'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file,
-                         htmllint_ignore: typed_list(str)=()):
```

Comment:
I am not sure but I think you should do this in a separate commit, isnt exactly the issue you are resolving here :) You need to add a commit body too :)

Reply:
Some of the tests aren't passing if this change isn't made.Previously htmllint_ignore acts as a tuple if no argument is passed and is typecasted into a list when arguments are passed which is creating problems in order to append the same.

Reply:
Ok... That fix can be merged before the rest, @Makman2 your thoughts ?

Reply:
But should there be a problem putting the change in a different commit?

Reply:
In the same issue two different commits or should I create another issue?

Reply:
two commits with each linked to a different issue, but wait for Makman2 to weigh in on this

Reply:
imo that's fine, it's a consequence to use a list instead of a tuple to implement the desired features. In fact it's better to use a tuple to make the default value immutable, however this isn't really practical here, we have to use a list.